configure.scss

```diff
@@ -1,20 +1,20 @@
 $_n-ui-features: (
 	ads: true,
 	buttons: true,
-	grid: true,
-	// always in silent mode
-	colors: false,
-	// always in silent mode
-	icons: false,
-	typography: true,
+	card: false,
 	cookieMessage: false,
-	message-prompts: false,
+	expander: false,
+	footer: true,
+	forms: false,
+	grid: true,
+	header: true,
+	messagePrompts: false,
 	myft: false,
 	notification: false,
-	welcomeMessage: false,
-	header: true,
-	footer: true
-);
+	overlay: false,
+	typography: true,
+	welcomeMessage: false
+) !default;
```

Comment:
Just in case this is accidentally included again which would mean this assignment would overwrite the configured state.

We could also throw a wobbly.